lib/nerves_time/rtc.ex

```diff
@@ -0,0 +1,130 @@
+defmodule NervesTime.RTC do
+  require Logger
+  use GenServer
+  use Bitwise
+
+  def update() do
+    GenServer.call(__MODULE__, {:set_time, NaiveDateTime.utc_now()})
+  end
+
+  def time() do
+    GenServer.call(__MODULE__, :get_time)
+  end
+
+  def start_link(_args) do
+    GenServer.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  defp set_time(%NaiveDateTime{} = time) do
```

Comment:
Nit: I'd pull `set_time/1` out to a utilities or system module on the next pass since it doesn't feel like part of any of this.